plugin.yaml

```diff
@@ -10,62 +10,318 @@ plugins:
     # URL to archive containing the plugin or name of directory containing
     # the plugin if it is included in the the blueprint directory under the
     # "plugins" directory.
-    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/master.zip
+    source: https://github.com/szpotona/cloudify-gcp-plugin/archive/add_disk.zip
 
 node_types:
   cloudify.gcp.nodes.Instance:
     derived_from: cloudify.nodes.Compute
+    properties:
+      use_external_resource:
+        description: >
+          Indicate whether the resource exists and use existing (true) 
+          or if Cloudify should create new resource (false).
+        type: boolean
+        default: false
+      gcp_config:
+        description: >
+          A dictionary of values to pass to authenticate with the GCP API.
+        default: {}
+      image_id:
+        description: >
+          The ID of the image in your GCP account.
+        type: string
+        default: {}
+      instance_type:
+        description: >
+          The instance's type. All available instance types can be found here: 
+          https://cloud.google.com/compute/docs/machine-types
+        type: string
+        default: n1-standard-1
+      instance_additional_settings:
+        description: >
+          Additional instance settings.
+        default: {}
     interfaces:
       cloudify.interfaces.lifecycle:
         start:
```

Comment:
seems like this PR is based on old code, because other PRs (https://github.com/cloudify-cosmo/cloudify-gcp-plugin/pull/6) contain modification not found here (e.g. mapping start to create_instance was fixed there). correct?